### PR TITLE
Added store selector to menu footer(#2e5w2hb)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
 

 Closes #

 ### Short Description and Why It's Useful
 

 ### Screenshots of Visual Changes before/after (If There Are Any)
 

 **IMPORTANT NOTICE** - Remember to add changelog entry

 ### Contribution and Currently Important Rules Acceptance
 

 - [X] I read and followed [contribution rules](https://github.com/hotwax/threshold-management#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/170970998-07c72339-3a08-4db3-8815-379687547910.mp4

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/170973901-6a9fabaa-24fb-47a5-babe-e6a842d32fd0.mp4

File: src/views/SelectProduct.vue [hunk 1]
```diff
@@ -231,6 +232,12 @@ export default defineComponent({
       queryString: ''
     }
   },
+  mounted() {
+    emitter.on("productStoreChanged", this.getProducts);
+  },
+  unmounted() {
+    emitter.on("productStoreChanged", this.getProducts);
```
Comment (ymaheshwari1) [inline src/views/SelectProduct.vue:239]:
```suggestion
    emitter.off("productStoreChanged", this.getProducts);
```

File: src/views/ThresholdUpdates.vue [hunk 2]
```diff
@@ -326,8 +327,12 @@ export default defineComponent({
     })
   },
   mounted(){
+    emitter.on("productStoreChanged", this.refreshJobs);
     this.store.dispatch('util/getServiceStatusDesc')
   },
+  unmounted(){
+    emitter.on("productStoreChanged", this.refreshJobs);
```
Comment (ymaheshwari1) [inline src/views/ThresholdUpdates.vue:334]:
```suggestion
    emitter.off("productStoreChanged", this.refreshJobs);
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/SelectProduct.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/SelectProduct.vue]: I have updated it sir.

File: src/views/ThresholdUpdates.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/ThresholdUpdates.vue]: I have updated it sir.

**Comment by ymaheshwari1** [comment]
Closing as handled in #171

**Pull request closed without merging**